src/components/Badge/Badge.tsx

```diff
@@ -1,35 +1,17 @@
 import React from 'react';
 import classNames from 'classnames';
+import { Clickable } from '../Clickable';
 
 import './Badge.scss';
 
-export const TRENDING_THEME = 'trending';
-export const BESTSELLER_THEME = 'bestseller';
-export const FALL_ESSENTIAL_THEME = 'fall-essential';
+type Theme = 'bestseller' | 'trending' | 'fall-essential' | 'fall-25';
 
-const THEMES = {
-  [BESTSELLER_THEME]: 'Bestseller',
-  [TRENDING_THEME]: 'Trending',
-  [FALL_ESSENTIAL_THEME]: 'Fall Essential'
-} as const;
-
-type Theme = keyof typeof THEMES;
-type CustomProps = { children: React.ReactNode; theme?: string }; //if a theme is passed in that is not an expected theme, treat this as a custom badge
```

Comment:
we don't want these custom props anymore?

Reply:
nah, we just select a theme and insert whatever text we want inside.